test/github_pullrequest_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/google/go-github/v66/github"
 	"github.com/openshift-pipelines/pipelines-as-code/pkg/opscomments"
 	tgithub "github.com/openshift-pipelines/pipelines-as-code/test/pkg/github"
+
```

Comment:
?

Reply:
gofmt gofumpt goimports goimports-reviser, one of those 🤷🏻